homeassistant/components/light/lifx.py

```diff
@@ -30,9 +32,10 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-REQUIREMENTS = ['aiolifx==0.6.3', 'aiolifx_effects==0.2.1']
+DEPENDENCIES = ('lifx',)
```

Comment:
We normally use a list here.

Reply:
Oh, I thought that looked odd. (It was copied from the Cast migration but I see now that this is not the normal way.)